tests/examples/authors_combined_terms.example

```diff
@@ -0,0 +1,72 @@
+
+** New syntax rules, which can be directly communicated to users in interface **
+Comma:
+ - Match part before the comma to surname AND match part after the comma to
+   forename (or as prefix) on same author.
+No comma:
+ - match all parts across all authors with simple AND
+Semicolon:
+ - Match all terms before semicolon on same author, after semicolon on different author, etc.
+No semicolon:
+ - Match all terms across all authors.
+Quotation marks:
+ - "Luke" "Luke Edholm" "L Edholm" "Edholm", "Edholm, L." -- literal on any
+   author name field, except continue to support commas.
+
+New author syntax examples:
+Edholm, Luke => results for any paper with: Surname == "Edholm" AND forename == "Luke" on -same person-
+Edholm, L => results for any paper with: surname == "Edholm" AND (forename begins with L or == L) -same person- (infer that this is just one person)
+Edholm; Luke => results for any paper with "Edholm" in a name field AND "Luke" in a separate name field. (indication of two separate persons)
+Edholm, Luke; Smith, Jane => (surname == "Edholm" AND forename matches or prefix "Luke") AND (surname == "Smith", forename matches or prefix "Jane")
+
+** Single author, initial and/or first name **
+Previously, or implied from user feedback expectations:
+  L Edholm => results for L Edholm, Luke Edholm
+  Luke Edholm => results for Luke Edholm, L Edholm (and not Luke * or * Edholm)
+New method:
+  L Edholm => results for any paper containing 'Edholm' in an author name AND 'L'
+              in an author name. Boost: results containing 'L Edholm' exactly.
+  Luke Edholm => results for any paper containing 'Edholm' in an author name AND 'Luke'
+              in an author name. Would be the same as searching for multiple authors
+              without additional syntax.
+
+Kuo-Wei Huang => results for Kuo-Wei Huang (currently only KuoWei without hyphen works)
+Volker Eyert => results for Volker Eyert (may be missing index issue?) including 1607.01703
+Weiwei Wu => results for Weiwei Wu (as Luke Edholm example above)
+Juven Wang => results for Juven Wang (as Luke Edholm example above)
+
+
+** Single author multiple surnames **
+Alvarez Cartelle => results for 'Alvarez' AND 'Cartelle' somewhere in author field (can be separate people)
+Pena Alvarez => results for 'Pena' AND 'Alvarez' in author field
+Pena-Alvarez => results for 'Pena-Alvarez' in author field
+
+** Multiple authors denoted by space or other punctuation **
+waalewijn tackmann => results for papers by 'Waalewijn' AND 'Tackmann'
+runkel gainutdinov => results for papers by 'Runkel' AND 'Gainutdinov'
+Ceballos Ziegler => results for papers with authors 'Ceballos' AND 'Ziegler' (with or without other authors)
+Ceballos, Santos, and Ziegler => old system: results for "Ceballos, Santos, Ziegler"
+                                 new system: invalid syntax (should consider case when more than one comma appears). Would this parse as "Ceballos, Santos" AND "Santos, and Ziegler"
```

Comment:
In https://github.com/cul-it/arxiv-search/pull/177, this would be interpreted as surname: ``Ceballos``, forename: ``Santos and Ziegler``. There is really only so much we can do to guess what they user wants.

Reply:
``Ceballos, Santos; Ziegler, Rowena`` => (surname: ``Ceballos``, forename: ``Santos``) AND (surname: ``Ziegler``, forename: ``Rowena``)